v2/internal/plugins/collector/proxy/manager.go

```diff
@@ -384,6 +384,18 @@ func (cm *ContextManager) DefineGroup(name string, description string) {
 	cm.groupsDescription[name] = description
 }
 
+func (cm *ContextManager) SetAllowDynamicLastElement() {
```

Comment:
would remove this one and set this behaviour as default

Reply:
done